test/e2e/dra/dra.go

```diff
@@ -773,6 +775,58 @@ var _ = ginkgo.Describe("[sig-node] DRA [Feature:DynamicResourceAllocation]", fu
 		multipleDriversContext("using only drapbv1alpha3", false, true)
 		multipleDriversContext("using both drapbv1alpha2 and drapbv1alpha3", true, true)
 	})
+
+	// This test covers aspects of non graceful node shutdown by DRA controller
+	// More details about this can be found in the KEP:
+	// https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/2268-non-graceful-shutdown
+	ginkgo.Context("cluster", func() {
```

Comment:
The "cluster" context itself isn't about non-graceful shutdown. This comment belongs in front of the `ginkgo.It` with the test.

Regarding context: can you put the `It` under 
```
ginkgo.Context("multiple nodes", func() {
		nodes := NewNodes(f, 2, 8)
		ginkgo.Context("with network-attached resources", func() {
			driver := NewDriver(f, nodes, networkResources)
			b := newBuilder(f, driver)

                        ginkgo.It(...
```

There's no need for a new context. The advantage is that tests that use a similar setup are also grouped together and there's less code duplication.

Reply:
done, please review again